docs/INSTALL.md

```diff
@@ -265,6 +265,29 @@ This will generate a `config.json` file with the proper information and use the
 
 The `csutil` tool uses the Crownstone BLE library under the hood. That means that you can also setup hardware that is not attached to your machine. In that case make sure you have the right config values set (e.g. in `CMakeBuild.runtime.config`).
 
+## Buiding on  Archlinux
+
+It is recommended to use a virtual machine (running Ubuntu ubuntu 18.04 or newer) for compiling/developing with Bluenet in archlinux.
+
+The user of the host machine has to be added to the vboxusers group. Log out and back in after running this command.
+
+    sudo gpasswd -a m vboxusers
+
+Also it is recommended to check if a J-Link programmer is correctly connected on the host machine.
+    
+    lsusb | grep J-Link
+    
+Running these commands guest editions have to be installed on the Ubuntu virtual machine. Insert the guest editions into the VM by going to the bar menu and pressing Devices > Insert guest editions. Ubuntu should show a pop up to help you further.
```

Comment:
"Running these commands guest editions have to be installed" is this a typo?

Reply:
You could make it `Devices > Insert guest editions` to clarify what part is the "command".